src/k8s/request.js

```diff
@@ -196,7 +200,13 @@ const setNetworks = (vm, basicSettings, networks) => {
 };
```

Comment:
why are we deleting disks and volumes at https://github.com/kubevirt/web-ui-components/pull/55/files#diff-b6fca8acbec69ecc7320bf2aa4787625R195 when using PXE?

Reply:
This is just temporary as we do not have Storage tab yet. I need to make sure that there is no disk/volume when PXE is used as it might cause troubles. The disk/volume deletion should be removed once Storage Tab is merged.

Reply:
ok, I removed it in #10